OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/fragments/SkinPlayerFragment.java

```diff
@@ -0,0 +1,98 @@
+package com.ooyala.sample.fragments;
```

Comment:
This will be a little strange, but would you mind moving this to the "utils" package?  I know it's named strange, but it is generally all of the files that are not the player 

This allows us to easily copy all files into the big completeSampleApp in the future, if desired